WordPress/UITestsFoundation/Globals.swift

```diff
@@ -6,6 +6,17 @@ import ScreenObject
 // than stored as a reference? 🤔
 public let navBackButton = XCUIApplication().navigationBars.element(boundBy: 0).buttons.element(boundBy: 0)
 
+// Sometimes the Back Button in Navigation Bar is not recognized by XCUITest as an element.
+// This method identifies when it happens and uses a swipe back gesture instead of tapping the button.
+public func navigateBack() {
+    if XCUIApplication().navigationBars.element(boundBy: 0).buttons.count == 1 {
+        let leftEdge = XCUIApplication().coordinate(withNormalizedOffset: CGVector(dx: 0, dy: 0.5))
+        let center = XCUIApplication().coordinate(withNormalizedOffset: CGVector(dx: 0.5, dy: 0.5))
+
+        leftEdge.press(forDuration: 0.01, thenDragTo: center)
+    } else { navBackButton.tap() }
```

Comment:
I wonder if that's got something to do with my hypothesis that storing elements in `let` instead of computing them at runtime may result in them not being found? 🤔 

See comment in `navBackButton` above: https://github.com/wordpress-mobile/WordPress-iOS/blob/2fc79960c7febaeb1d84051b99928b588cac7bf9/WordPress/UITestsFoundation/Globals.swift#L5-L7

If that was the case, then we could try to compute `navBackButton` instead, which would keep the code simpler:

```swift
public var navBackButton: XCUIElement { XCUIApplication().navigationBars.element(boundBy: 0).buttons.element(boundBy: 0) }
```

Reply:
I can change it and give it a try but for this specific issue I'm pretty sure that test code is not the problem. Accessibility Inspector is not able to identify the button when the issue kicks in and the issue will be gone if you just leave the page and get back to it. That's why I believe it is on the Simulator side. @jostnes also mentioned seeing this happening in previous projects.